scripts/bulletActions/bulletActions.gml

```diff
@@ -0,0 +1,16 @@
+function fireBullet() {
+	if (!obj_player.bullet_fired) {
+		var inst = instance_create_layer(x - 32, y - 64, "Instances", Dot);
+		inst.depth -= -1;
```

Comment:
inst.depth += 1; would probably read a bit nicer but that's mostly ant fuckery ;)